src/main/java/uk/gov/hmcts/reform/professionalapi/controller/S2sClient.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.professionalapi.controller;
+
+import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+import com.google.common.collect.ImmutableMap;
+import com.warrenstrange.googleauth.GoogleAuthenticator;
+
+import io.restassured.RestAssured;
```

Comment:
Correct me if I am wrong here. I think RestAssured is used for testing REST services right ?

Reply:
I believe you are correct, the class has been removed in this commit: https://github.com/hmcts/rd-professional-api/pull/910/commits/c30848e3bc7dbe2db43a99dae26ffd39a2ecc9cf